build.rs

```diff
@@ -115,7 +115,13 @@ fn build_c_code(out_dir: &str) -> Result<(), std::env::VarError> {
             format!("CMAKE_BUILD_TYPE={}", cmake_build_type),
             format!("BUILD_PREFIX={}/", out_dir),
         ];
-        run_command_with_args(&"make", &args);
+
+        // If `MAKE_COMMAND` is set, we will use the env target to build with.
+        // Otherwise, we will default to using `make`.
+        match env::var("MAKE_COMMAND") {
+            Ok(command) => run_command_with_args(&command, &args),
+            _ => run_command_with_args(&"make", &args),
```

Comment:
I'm not 100% certain if this is idiomatic--both stylistically and purpose-wise. Some languages are very specific in their stylistic choices, so I don't know if that needs to be tabbed and matched with the other "=>".

Moreover, I'm not 100% sure if there's a better way of handling errors than matching, but this seems a little heavy handed.